src/ApiCompat/Microsoft.DotNet.ApiCompat.Shared/CommonResources.resx

```diff
@@ -117,12 +117,18 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="BreakingChangesFound" xml:space="preserve">
+    <value>Either fix breaking change, or update compatibility suppression files, rebuild with \p:GenerateCompatibilitySuppressions or --generate-suppression-file.</value>
+  </data>
   <data name="CreateWorkItemPerAssemblyAssembliesNotEqual" xml:space="preserve">
     <value>When enqueuing work items per assembly, the number of passed in left and right assemblies must be equal.</value>
   </data>
   <data name="InvalidRexegStringTransformationPattern" xml:space="preserve">
     <value>Invalid transformation pattern provided: '{0} - {1}'.</value>
   </data>
+  <data name="NoBreakingChangesFound" xml:space="preserve">
+    <value>Package validation was successful. No breaking changes.</value>
```

Comment:
This isn't just about Package Validation. The message will also be printed when assemblies are being compared. I would rephrase to `APICompat ran successfully without finding any breaking changes.`.